psi4/driver/json_wrapper.py

```diff
@@ -98,6 +98,7 @@ def run_json(json_data, clean=True):
     if return_output:
         with open(outfile, 'r') as f:
             json_data["raw_output"] = f.read()
+        core.close_outfile() # On Windows the file has to be closed before deleting
```

Comment:
Is it a successful json run or a failed json run that you're getting the dangling outfile for? There's two sets of outfile handling going on (or rather there's a set-and-a-half at present). The lines 99–100 one should be sound, as the `with open` should close the outfile under all circumstances. I'm guessing it's the psi handling one starting [here](https://github.com/psi4/psi4/pull/1180/files#diff-f0c7d86da302944bdf61cc5d48e3701cR69) that's causing trouble that you're trying to fix by the new line 101?

Linux doesn't seem to like the addition.

Reply:
Weird, it worked locally on Linux.

Reply:
I've restarted one of the travis jobs to see if it was a glitch.

Reply:
No glitch. I'll try it locally.

```
The following tests FAILED:
	401 - json-v11-energy (Failed)
  <<<  CTest complete with status 8.  >>>
The command "python ../.scripts/travis_run_test.py" exited with 0.
0.05s$ python ../.scripts/travis_print_failing.py
  <<<  Failing outputs follow.  >>>
json-v11-energy failed. Here is the output:
No JSON output file found
The command "python ../.scripts/travis_print_failing.py" exited with 8.
51.04s$ ./stage/${TRAVIS_BUILD_DIR}/Install/bin/psi4 --test
============================= test session starts ==============================
platform linux -- Python 3.6.6, pytest-3.7.2, py-1.5.4, pluggy-0.7.1 -- /home/travis/miniconda/envs/p4env/bin/python
cachedir: .pytest_cache
rootdir: /home/travis/build/psi4/psi4/build, inifile:
collected 22 items                                                             
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_gdma PASSED [  4%]
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_mrcc SKIPPED [  9%]
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_chemps2 SKIPPED [ 13%]
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_dftd3 PASSED [ 18%]
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_libefp LIBEFP: parameters for fragment "NH3" are already loaded
PASSED [ 22%]
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_pcmsolver SKIPPED [ 27%]
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_erd SKIPPED [ 31%]
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_simint PASSED [ 36%]
stage/home/travis/build/psi4/psi4/Install/lib/psi4/tests/test_addons.py::test_json /home/travis/.travis/job_stages: line 78: 15722 Segmentation fault      (core dumped) ./stage/${TRAVIS_BUILD_DIR}/Install/bin/psi4 --test
```

Reply:
Not a glitch. The v11 test cases are unhappy with the new line for me on Linux. If I move the new line around, other json tests are unhappy. I'm confused at this point about who's closing the output. @dgasmith, what do you say to just dropping the [`unlink` line](https://github.com/psi4/psi4/pull/1180/files/366c4e4012b1d3ac948061b646d6001916d0d047#diff-f0c7d86da302944bdf61cc5d48e3701cL101)? `return_output` key, after all, is about what's in the json file. Hopefully Windows will be content with that.

Reply:
Thanks, I'll try to look at this again.